vue-components/src/components/Link.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+	<span
+		:class="[
+			'wikit',
+			'wikit-Link',
+			(underlined == true) ? `wikit-Link--underlined` : ''
+		]"
+	>
+		<Icon
+			v-if="icon === 'before'"
+			class="wikit-Link__icon wikit-Link__icon--before"
+			type="link"
+			color="progressive"
+			size="large"
+		/>
+		<span class="wikit-Link__content"><slot /></span>
+		<Icon
+			v-if="icon === 'after'"
+			class="wikit-Link__icon wikit-Link__icon--after"
+			type="newwindow"
+			color="progressive"
+			size="large"
```

Comment:
```suggestion
			size="xsmall"
```

Reply:
Figma says it should be large. Am I missing something really obvious? https://www.figma.com/file/gi3JYwSUQYhKS0mHwK25cQwi/WiKit?node-id=3922%3A5

Reply:
Mind if we use the xsmall size in this story? 

I've been hesitating to pass the UI tests only because of this sizing dilemma, but I don't think it should be a blocker at this point: should we apply a different, smaller size to icons placed after text elements just because of aesthetic reasons? We have at least two cases: labels and links, but it keeps feeling arbitrary.

Reply:
I only documented the --before icon in Figma :-/

Reply:
Hey, @Ladsgroup. After further discussion with Charlie, we agreed on the fact that (like with Labels) the suffix icon shouldn't be so big, as it gets distracting while its purpose is providing secondary information rather than making the link recognizable/communicating meaning. So I committed some changes to:

- Introduce the sizes 'small' and 'xsmall' of the Icon component
- Separate the size tokens of the prefix and suffix in Link.json
- Apply a 'xsmall' size to the --after icon

Hope that's fine!